src/app/Components/Toast/types.ts

```diff
@@ -1,6 +1,8 @@
 import { ActionSheetProps } from "@expo/react-native-action-sheet"
 import { Color, IconProps } from "palette"
 
+export type ToastDuration = "LONG" | "SHORT"
```

Comment:
```suggestion
export type ToastDuration = "long" | "short"
```

Let's keep it consistent with other props.

Reply:
@olerichter00 What other props? Capitalizing duration denotes that it is a constant, and even if we are passing a string, anyone can read this and immediately understand that the string must have been mapped to a constant somewhere. In this case, where we are not using enums, uppercase strings conveys the same message. I think this is a standard naming convention in most languages (Or is it different in JS?)
Eg: [Java](https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html#:~:text=The%20names%20of%20variables%20declared,%2C%20for%20ease%20of%20debugging.)
Also similar convention used here in React Native: [ToastAndroid](https://reactnative.dev/docs/toastandroid)

Reply:
> Also similar convention used here in React Native: [ToastAndroid](https://reactnative.dev/docs/toastandroid)
Here the key is in capitalcase. The docs don't say anything about the content of it.

> Eg: [Java](https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html#:~:text=The%20names%20of%20variables%20declared,%2C%20for%20ease%20of%20debugging.)
We're using TypeScript ;)

>What other props?
For example [here](https://github.com/artsy/eigen/blob/681d9a06058c6020161ca5427300150327788cd8/src/app/Components/Toast/types.ts#L4) and [here](https://github.com/artsy/eigen/blob/681d9a06058c6020161ca5427300150327788cd8/src/palette/Theme.tsx#L24) and pretty much everywhere in the codebase.

Reply:
> For example [here](https://github.com/artsy/eigen/blob/681d9a06058c6020161ca5427300150327788cd8/src/app/Components/Toast/types.ts#L4) and [here](https://github.com/artsy/eigen/blob/681d9a06058c6020161ca5427300150327788cd8/src/palette/Theme.tsx#L24) and pretty much everywhere in the codebase.

😄 I see. @olerichter00 in the examples you have listed here, these props are used as is - i.e as the strings that they are or they map to another string, so it makes sense to use them as plain strings and they do not map to a fixed number constant. But by convention, when you map a string(key) to a final number constant you definitely should uppercase the name so it is obvious to everyone reading it, else to me there is no way I will ever think that "long" maps to a number.  It is not just a java thing. 
For eg: https://github.com/facebook/react-native/blob/main/Libraries/Components/ToastAndroid/NativeToastAndroid.js#L15

Reply:
You can also use "long" as a key, I don't see any issue with that and would prefer to keep the API of the components consistent.

@MounirDhahri what do you think?

Reply:
Will change it 👍 . Consistency is more important here I guess. A mixture of upper and lowercased props all over the place doesn't look clean. @olerichter00

Reply:
Thanks a lot 🙇